TechTable/index.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+//index.php
+
+?>
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>IIITD|TechTable</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
+  <script src="https://cdn.datatables.net/1.10.12/js/jquery.dataTables.min.js"></script>
+  <script src="https://cdn.datatables.net/1.10.12/js/dataTables.bootstrap.min.js"></script>  
+  <link rel="stylesheet" href="https://cdn.datatables.net/1.10.12/css/dataTables.bootstrap.min.css" />
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+  <style>
+    /* Styling for datatable */
+  .box
+  {
+   max-width:600px;
+   width:100%;
+   margin: 0 auto;;
+  }
+  /* For padding above and below the header and footer respectively*/
+  body{        
+        padding-top: 0px;
+        padding-bottom: 0px;
+    }
+  .header{
+        width: 100%;
+        background: #ffffff;
+        padding: 10px 10px;
+        color: #ffffff;
+        top: 0;
+    }
+    .footer{
+        width: 100%;
+        background: #808080;
+        padding: 10px 10px;
+        color: #ffffff;
+        bottom: 0;
+    }
+    
+        
+  </style>
+ </head>
+ <body>
+  <div class="header">
+        <div class="container">
+                <a href="https://iiitd.ac.in/"><img src="IIITDLogo.jpg" alt="Home" align="left"></a>
+                <img src = "TechTable.png" alt="TechTable" width="200" height="100" align="right">
+        </div>
+    </div>
+
+  <div class="container">
+   <br />
+   <h2 align="center">TechTable</h2>
+   <br />
+   <div style="clear:both"></div>
+   </form>
+   <div class="table-responsive">
+    <table class="table table-striped table-bordered" id="data-table">
+     <thead>
+      <tr>
+       <th>Course Name</th>
+       <th>Course Acronym</th>
+       <th>Course Code</th>
+       <th>Prerequisites</th>
+       <th>Antirequisites</th>
+       <th>Semester</th>
+      </tr>
+     </thead>
+    </table>
+   </div>
+
+   <div class="footer">
+        <div class="container">Copyright &copy; 2019 IIITD</div>        
+    </div>
+ </body>
+</html>
+<script>
+
+$(document).ready(function(){
+  event.preventDefault();
```

Comment:
This line isn't needed, remove it.

`preventDefault` is useful when you need to block browser's default behavior, e.g. you'd use it when you want to stop the default action of submitting the form when user clicks the submit button. In a `ready` event, it isn't really needed.

Reply:
Yes, `preventDefault` was being used initially when we were selecting the csv files to be uploaded from our systems for testing. It has been removed now.